EssentialFeed/EssentialFeediOS/Feed UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ImageCommentCellController.swift
+//  EssentialFeediOS
+//
+//  Created by Adrian Szymanowski on 16/03/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCellController {
+	private var cell: ImageCommentCell?
+	private let viewModel: () -> ImageCommentViewModel
```

Comment:
This property shouldn't be a clousre. It can be just a `ImageCommentViewModel`:

```suggestion
	private let viewModel: ImageCommentViewModel
```